src/components/apps/queries.js

```diff
@@ -43,17 +43,20 @@ export function useAppElementInfoTypes(enabled, onSuccess, onError) {
     });
 }
 
+export function sortReferenceGenomes(genomes) {
+    return stableSort(genomes, (a, b) => a.name.localeCompare(b.name));
+}
+
 export function useReferenceGenomes(enabled, onSuccess, onError) {
     return useQuery({
-        queryKey: [REFERENCE_GENOMES_QUERY_KEY, { deleted: false }],
```

Comment:
Do we want the deleted genomes as well ? or just the service returns genomes that not deleted by default ?

Reply:
Ahh nm, i see you updated the key below.